public/locales/en/iTwinMobileUI.json

```diff
@@ -3,6 +3,7 @@
     "99+": "99+"
   },
   "modal-entry-form": {
+    "restricted-characters": "Field cannot contain following symbols: ",
```

Comment:
You must never construct strings in code that include localizations. In this case in particular, it is possible that some languages would not have the variable portion be at the end. So, you need to use something like so:

```json
    "restricted-characters": "Field cannot contain following symbols: {{symbols}}"
```

Then later when you look up the localized string, you add the following as a second argument to `MobileUi.translate`:
```ts
{ symbols: fields[index].restrictedCharacters }
```